specs/eip4844/polynomial-commitments.md

```diff
@@ -98,22 +97,14 @@ def reverse_bits(n: int, order: int) -> int:
     return int(('{:0' + str(order.bit_length() - 1) + 'b}').format(n)[::-1], 2)
 ```
 
-#### `bit_reversal_permutation_scalars`
+#### `bit_reversal_permutation`
 
 ```python
-def bit_reversal_permutation_scalars(l: Sequence[BLSFieldElement]) -> Sequence[BLSFieldElement]:
+def bit_reversal_permutation(l: Sequence[T]) -> Sequence[T]:
     """
-    Apply the bit-reversal permutation to a list. This operation is idempotent.
-    """
-    return [l[reverse_bits(i, len(l))] for i in range(len(l))]
-```
+    Return a copy with bit-reversed permutation. This operation is idempotent.
```

Comment:
p.s. I changed this docstring because the old version sounds like this function would change the input with side effects.